packages/salesforcedx-utils/src/cli/forceConfigGet.ts

```diff
@@ -10,7 +10,8 @@ import { CommandOutput } from './commandOutput';
 import { SfdxCommandBuilder } from './sfdxCommandBuilder';
 /**
  * @deprecated
- * NOTE: This code is deprecated in favor of using ConfigUtil.ts
```

Comment:
I'm not sure why I deleted this NOTE honestly.  Probably need to visit if this command is necessary.

Reply:
ok..yeah I'll put it back.  This is only used in the debuggers currently that can't access configUtils in the vscode-utils package FYI.